bosi/rhosp_resources/build_scripts/rhosp_packager.sh

```diff
@@ -2,16 +2,45 @@
 
 # Following build params expected for this script:
 # OpenStackBranch
-# BosiBranch
-# RHOSPVersion
+# BcfBranch
 # Revision
+# IvsBranch (optional)
+
+# mapping for OpenStackBranch to RHOSPVersion, default is latest = 9
+# occasionally cleanup when we stop supporting certain versions
+RHOSPVersion="9"
+case "$OpenStackBranch" in
+  *"mitaka"*) RHOSPVersion="9" ;;
+  *"liberty"*) RHOSPVersion="8" ;;
+  *"kilo"*) RHOSPVersion="7" ;;
+esac
+
+# if IvsBranch is not specified, it is same as BcfBranch
+if [ -z  $IvsBranch  ]
+then
+    IvsBranch="$BcfBranch"
+fi
+
+# if branch is not master, append 'v' e.g. v3.7.0
+if [ $IvsBranch != "master" ]
```

Comment:
so ivs is 1:1 with bcf, so that wouldn't need version magic.
for other openstack packages, it would be dynamic, since we don't know the last bug-fix-id bit. so we get the package version number from the rpm name.